DNN Platform/DotNetNuke.ModulePipeline/ModuleControlPipeline.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.IO;
+using DotNetNuke.Entities.Modules;
+using DotNetNuke.Instrumentation;
+using DotNetNuke.Services.Localization;
+using DotNetNuke.UI.Modules.Html5;
+using DotNetNuke.UI.Modules;
+using DotNetNuke.Web.Razor;
+using System.Collections.Generic;
+using DotNetNuke.Web.Mvc;
+
+#if NET472
+using System.Web.UI;
+#endif
+
+namespace DotNetNuke.ModulePipeline
+{
+
+    public class ModuleControlPipeline
+#if NET472
+        : IModuleControlPipeline
+#endif
```

Comment:
For future support can we just add a comment here as to why this compiler directive was needed

Reply:
I wasn't sure if we wanted to add comments in to explain the reasoning behind this or not. Considering the nature of OSS it is probably best to create the comment in code.